lldb/test/Shell/CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 add_custom_target(lldb-shell-test-deps)
 set_target_properties(lldb-shell-test-deps PROPERTIES FOLDER "LLDB/Tests")
 add_dependencies(lldb-shell-test-deps lldb-test-depends)
+if(LLDB_BUILT_STANDALONE)
+  get_target_property(CLANG_RESOURCE_DIR clang-resource-headers INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
In `LLDBStandalone.cmake` we already compute the clang resource dir and put it into the variable `LLDB_EXTERNAL_CLANG_RESOURCE_DIR`. You could probably just reference that variable instead of recalculating it here.